### PR TITLE
Enable CentOS 9 builds and add Testing Farm e2e tests

### PR DESCRIPTION
**Pull request by mmartinv**
- **feat: enable CentOS/Fedora ELN builds**
- **feat: enable e2e testing farm tests**
- **fix: lock clap_builder to 4.4 version**

Fixes: #646

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -15,34 +15,41 @@ copy_upstream_release_description: true
...
+          fedora-developement: {}
+          fedora-eln:
+              additional_repos:
+                  - https://kojipkgs.fedoraproject.org/repos/eln-build/latest/$basearch/
+
+    - job: tests
+      trigger: pull_request
+      identifier: onboarding
+      fmf_path: test/fmf
+      tmt_plan: plans/onboarding
```
Comment (henrywang) [inline .packit.yaml:44]: The tmt plan defined here, so the rpmlint will not be run in this case.

Comment (mmartinv) [reply .packit.yaml:44]: I removed it from the source code, we will probably add more tests in the future.

**Review by henrywang — APPROVED** [review]
Looks good to me. Thanks!

**Review by 7flying — APPROVED** [review]
LGTM thanks!

**Review by henrywang — APPROVED** [review]

**Comment by henrywang** [comment]
Thanks @mmartinv !

**Pull request merged**